Fix crash when 'requests' is not available for import

RobinBoers opened the pull request:
Great project :)

I found a few small issues while trying to run it, which I fixed on my branch.

Also, for me applying the patches fails, I get a huge `.rej` file containing basically all changes. I think it's related to the fact that you're trying to patch the upstream `app.js`, while I'm using a documented fork by @TheSunCat.

RobinBoers committed:
Opportunistically import 'requests'

RobinBoers committed:
Add 'assets' to .gitignore

RobinBoers committed:
Make scripts executable

RobinBoers committed:
Ignore patch junk

RobinBoers committed:
Improve README formatting

amir16yp commented:
@RobinBoers ignore the bot i forgot i added this to my github lol
but yeah i should probably have an option to choose between upstream and a documented fork

amir16yp commented:
> @RobinBoers ignore the bot i forgot i added this to my github lol but yeah i should probably have an option to choose between upstream and a documented fork

Huh. I can just include the documented code in patch form instead of doing that, incase you want to start from scratch

RobinBoers commented:
Nevermind, I fixed it. `prettier` was silently not formatting the file (because of the `.gitignore` that I added), which caused the patch to fail. #2 should fix it.
